### ISSUE TITLE
Why there is a huge gap between the accuracy of training and testing?

### ISSUE THREAD

**Issue by lidongliang666** [liuch37/sar-pytorch#4]
why the accuracy is so poor in testing? Is it normal?

**Comment by lidongliang666** [comment]
![image](https://user-images.githubusercontent.com/26034924/101325022-0f753c00-38a6-11eb-8ff0-a13f6b921d2a.png)
比如,这在测试时,正确率基本为0,我很困惑.

**Comment by liuch37** [comment]
This is due to the size of training set in SVT is very small. We can expect to see super overfitting in its test set performance.

**Comment by lidongliang666** [comment]
非常感谢你及时的回复.我明白了,由于第一次遇到这种情况,于是没读代码就来询问了,我感觉抱歉.
